### DISCUSSION TITLE
What is the probability that the remaining marble from the same bag is also white?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#455] [Galaxy Trivia] [answered]
There are three bags, each containing two marbles. Bag A contains two white marbles, Bag B contains two black marbles, and Bag C contains one white marble and one black marble. You pick a random bag and take out one marble, which is white.

**Comment by ParanoidUser** [comment]
@wfrsk, @BrianMasiga, @awesomelewis2007, @DrakonZZZ maybe? 💰💰💰

**Reply by BrianMasiga** [reply]
The probability that the remaining marble from the same bag is also white depends on which bag the white marble was picked from.

Let's consider each bag individually:

Bag A: Both marbles are white, so the probability of picking a white marble is 1. If one white marble has been picked, the probability of picking another white marble is also 1.

Bag B: Both marbles are black, so the probability of picking a white marble is 0. There is no possibility of picking another white marble.

Bag C: There is one white marble and one black marble in the bag, so the probability of picking a white marble is 1/2. If a white marble has been picked, there is only one marble remaining, which is black.

Since we know that the marble picked was white, it must have come from either Bag A or Bag C. Therefore, the probability that the remaining marble from the same bag is also white is:

Bag A: 1
Bag B: 0
Bag C: 0

So, the overall probability is:

P(remaining marble is white) = P(A) * P(white from A) + P(C) * P(white from C) = (1/3) * 1 + (1/3) * (1/2) = 1/2

Therefore, the probability that the remaining marble from the same bag is also white is 1/2 or 50%.

**Reply by ParanoidUser** [reply]
**Request**: please don't post your answers as replies, as there is no way to mark any reply as an answer. Instead, post a new message to the thread. Thank you! 😊

**Comment by c8f888391789da0901167ee6134c4eb2** [comment]


**Reply by BrianMasiga** [reply]
The correct probability that the remaining marble from the same bag is also white is 1/2, not 2/3.
Here's the explanation:

We know that one of the white marbles was picked randomly from one of the three bags. There are now two possibilities:

The white marble was picked from Bag A: In this case, the remaining marble in Bag A is also white, so the probability of picking another white marble is 1.

The white marble was picked from Bag C: In this case, the remaining marble in Bag C is black, so the probability of picking a white marble is 0.

Since the probability of picking Bag A is 1/3 and the probability of picking Bag C is also 1/3, the overall probability of picking another white marble is:

(1/3 * 1) + (1/3 * 0) + (1/3 * 1/2) = 1/2

Therefore, the correct probability is 1/2, not 2/3.

**Comment by thuongtruong1009** [comment]
To solve this puzzle, we need to know a few things about the bag of marbles. Let's assume that the bag contains only two colors of marbles: white and black. Let's also assume that the number of white marbles in the bag is "w" and the number of black marbles is "b". Therefore, the total number of marbles in the bag is "w + b".

Now, suppose we randomly pick one marble from the bag without looking, and it turns out to be white. We are asked to find the probability that the remaining marble in the bag is also white.

To solve this problem, we can use conditional probability. Let's define two events:

A: Picking a white marble from the bag on the first draw.
B: Picking a white marble from the bag on the second draw.

We want to find P(B | A), which is the probability of event B occurring given that event A has occurred.

Using Bayes' theorem, we have:

P(B | A) = P(A | B) * P(B) / P(A)

where P(A | B) is the probability of picking a white marble on the first draw given that the second draw is also a white marble, P(B) is the probability of picking a white marble on the second draw without any prior knowledge, and P(A) is the probability of picking a white marble on the first draw.

P(A) can be calculated as:

P(A) = w / (w + b)

because there are w white marbles in the bag out of a total of w+b marbles.

Now, if we assume that we put the white marble we drew back into the bag, then the probability of picking a white marble on the second draw without any prior knowledge becomes:

P(B) = w / (w + b)

because the bag still contains w white marbles out of a total of w+b marbles.

Finally, we need to find P(A | B), which is the probability of picking a white marble on the first draw given that the second draw is also a white marble. This is simply:

P(A | B) = w / (w + b)

because if the second marble is also white, then there are w white marbles remaining in the bag out of a total of w+b marbles.

Putting it all together, we have:

P(B | A) = P(A | B) * P(B) / P(A)
= (w / (w + b)) * (w / (w + b)) / (w / (w + b))
= w / (w + b)

Therefore, the probability that the remaining marble in the bag is also white given that the first marble drawn was white is simply w / (w + b).

**Comment by 0x4248** [comment, marked as answer]
The probability of picking each bag is 1/3, since there are three bags and they are equally likely to be picked.

Given that you have picked a white marble, there are two possibilities: either you picked a marble from Bag A or you picked a marble from Bag C. The probability of each of these possibilities depends on the probability of picking each bag and the probability of picking a white marble from each bag.

The probability of picking a white marble from Bag A is 2/2 = 1, since all the marbles in Bag A are white. The probability of picking a white marble from Bag C is 1/2, since there is one white marble and one black marble in Bag C.

Using Bayes' theorem, we can calculate the probability of having picked Bag A given that we have picked a white marble:

P(A|W) = P(W|A) * P(A) / P(W)

where P(A|W) is the probability of having picked Bag A given that we have picked a white marble, P(W|A) is the probability of picking a white marble from Bag A, P(A) is the probability of picking Bag A, and P(W) is the probability of picking a white marble from any bag.

Plugging in the values, we get:

P(A|W) = (1 * 1/3) / ((1 * 1/3) + (1/2 * 1/3)) = 2/3

Therefore, the probability of having picked Bag A given that we have picked a white marble is 2/3.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/awesomelewis2007?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
